### PR TITLE
Remove patch

### PR DESCRIPTION
**Pull request by Owen1212055**
First time doing something like this, please make sure I didn't mess anything up. This drops the patch that was meant to fix https://bugs.mojang.com/browse/MC-68487 (as it has been fixed a while ago). 

image
Although, it also seems that this was edited in this patch, should this be reverted in a separate patch, edited this one, or just undone completetly? 

Feedback appreciated!

### CODE REVIEW

File: Spigot-Server-Patches/0362-Anti-Xray.patch [hunk 1]
```diff
@@ -1029,111 +1029,6 @@ index 0000000000000000000000000000000000000000..333763936897befda5bb6c077944d266
...
-         int k = 0;
-@@ -171,7 +203,7 @@ public class PacketPlayOutMapChunk implements Packet<PacketListenerPlayOut> {
- 
-             if (chunksection != Chunk.a && (!this.f() || !chunksection.c()) && (i & 1 << k) != 0) {
-                 j |= 1 << k;
--                chunksection.b(packetdataserializer);
-+                chunksection.writeChunkSection(packetdataserializer, chunkPacketInfo); // Paper - Anti-Xray - Add chunk packet info
-             }
-         }
-
```
Comment (JRoy) [inline Spigot-Server-Patches/0362-Anti-Xray.patch]: ????????

**Review by JRoy — CHANGES_REQUESTED** [review]

File: Spigot-Server-Patches/0362-Anti-Xray.patch (hunk 1 above)
Comment (Owen1212055) [reply Spigot-Server-Patches/0362-Anti-Xray.patch]: Yea... please give me a moment here haha.

**Comment by Draycia** [comment]
wat

**Comment by Owen1212055** [comment]
I didn't mean to close this. Anyways, I think I fixed everything up. I sincerely apologize for that, as I haven't really done this before.

**Comment by aikar** [comment]
This was done so intentionally and does not need to be changed.

**Pull request closed without merging**